packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -150,6 +156,21 @@ export class QuoteRequestor {
                 return false;
             }
 
+            if (order.takerAddress !== _opts.takerAddress) {
```

Comment:
Just in case there is a casing mismatch, it might be worthwhile to call `.toLowerCase()` on each of these

Reply:
Done, here: https://github.com/0xProject/0x-monorepo/pull/2574/commits/ed3d194c905c96924c79cea8fc3cab263d71287b